lib/chef/provider/aws_vpc_peering_connection.rb

```diff
@@ -0,0 +1,89 @@
+require 'chef/provisioning/aws_driver/aws_provider'
+require 'retryable'
+
+class Chef::Provider::AwsVpcPeeringConnection < Chef::Provisioning::AWSDriver::AWSProvider
+
+  def action_create
+    vpc_peering_connection = super
+
+    update_status(vpc_peering_connection, new_resource)
+  end
+
+  protected
+
+  def create_aws_object
+    options = {}
+    options[:vpc_id] = new_resource.vpc
+    options[:peer_vpc_id] = new_resource.peer_vpc
+    options[:peer_owner_id] = new_resource.peer_owner_id unless new_resource.peer_owner_id.nil?
+    options = AWSResource.lookup_options(options, resource: new_resource)
+
+    ec2_resource = new_resource.driver.ec2_resource
```

Comment:
I would like your opinion - how do you feel about the `ec2_client` and `ec2_resource` method names?